icons/src/outline.rs

```diff
@@ -104,6 +104,8 @@ pub enum Shape {
     ChatBubbleLeft,
     ChatBubbleOvalLeftEllipsis,
     ChatBubbleOvalLeft,
+    ChatPlus,
```

Comment:
I believe the changes to this file are from `dev`. I pulled `dev` and built it and got what look like the same changes as here. so I think it's OK to have the changes to icons here.